packages/create-bison-app/template/.github/workflows/main.js.yml.ejs

```diff
@@ -125,14 +125,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 <% if (host.name === 'vercel') { -%>
-      - uses: ngduc/vercel-deploy-action@master
+      - uses: amondnet/vercel-action@v20
         with:
           vercel-token: ${{ secrets.VERCEL_TOKEN }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
           vercel-org-id: ${{ secrets.VERCEL_ORG_ID}}
           vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID}}
           # SET THIS OR YOU WILL DEPLOY TO YOUR PERSONAL VERCEL NAMESPACE
-          # scope: ${{ secrets.VERCEL_TEAM_ID }}
+          # scope: ${{ secrets.VERCEL_ORG_ID }}
```

Comment:
Just for background, how will users know to uncomment this? Is it in the tutorial?

Reply:
It could definitely use some more documentation. I think the correct answer here is to prompt for this value when creating the project. I'll add a followup issue for that.